docs/GettingStartedDocs/GettingStarted.Windows.md

```diff
@@ -1,5 +1,5 @@
-Getting Started on Windows [Work in progress]
-=========================================
+Getting Started on Windows
```

Comment:
Don't remove the [Work in progress] label.

Reply:
ok